2024/door_02/hint.md

```
@@ -0,0 +1,5 @@
+- Check out https://color.adobe.com/search?q=christmas for some Christmas-themes color pallettes, any of which would be an improvement on the current theme. 😂
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)